### DISCUSSION TITLE
Add transparency grid behind site logo and icon

### DISCUSSION THREAD

**Discussion by leevigraham** [craftcms/cms#13210] [Ideas] [answered]
In cases where a white logo / icon is uploaded it's impossible to see the icon.

![image](https://github.com/craftcms/cms/assets/25124/4cd92c24-9b26-4849-af52-c8f7fe81aecd)

Maybe something like: 

![image](https://github.com/craftcms/cms/assets/25124/bdb3adc3-423d-4a6c-86c5-fd1ee02f9206)

**Comment by brandonkelly** [comment, marked as answer]
Craft 4.5.0 is out with that change!
